Source/WebKit/UIProcess/API/haiku/WebViewBase.h

```diff
@@ -54,8 +54,6 @@ class WebViewBase: public API::ObjectImpl<API::Object::Type::View>, public BView
     // hook methods
     virtual void FrameResized(float, float);
     virtual void Draw(BRect);
-    virtual void MouseDown(BPoint);
```

Comment:
do these changes belong to your other merge request?

Reply:
They were accidentally added in the previous pull request. This PR wouldn't build by itself without these removed.